pyFAI/app/average.py

```diff
@@ -69,8 +69,8 @@ def parse_algorithms(options):
             q2 = 1.0 - q1
         quantiles = (float(q1), float(q2))
 
-        if "quanties" not in methods:
-            logger.warning("Add quantiles to the set of methods as quantiles parameters is defined.")
+        if "quantiles" not in methods:
```

Comment:
This should fix a bug when `quantiles` was not in methods but when parameters were set. Quite unlikely to occur, I guess.